src/app/Loader.css

```diff
@@ -0,0 +1,83 @@
+.loader {
+	position: relative;
```

Comment:
Please use TailwindCSS :). Following for this .loader class is below...

@apply relative w-full min-height-[100vh] flex flex-col items-center justify-center;